.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -3,6 +3,13 @@ on:
   schedule:
     - cron: '*/5 * * * *'
   workflow_dispatch:
+  workflow_run:
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_run

Reply:
Can we remove the "schedule" event and rely on workflow_run only?

Reply:
Removed